core/themes/bartik/color/color.inc

```diff
@@ -86,6 +94,56 @@ $info = array(
         'titleslogan' => '#ffffff',
         'text' => '#3b3b3b',
         'link' => '#0073b6',
+        'menu' => '#f3f7fa',
+        'activemenu' => '#fefeff',
+      ),
+    ),
+    'giant_goldfish' => array(
+      'title' => t('Giant Goldfish'),
+      'colors' => array(
+        'top' => '#e0e4cc',
+        'bottom' => '#e0e4cc',
+        'bg' => '#ffffff',
+        'sidebar' => '#f3f4e9',
+        'sidebarborders' => '#ffffff',
+        'footer' => '#798053',
+        'titleslogan' => '#798053',
+        'text' => '#3b3b3b',
+        'link' => '#fa6900',
+        'menu' => '#798053',
+        'activemenu' => '#4c5037',
+      ),
+    ),
+    'black_drop' => array(
```

Comment:
What's the difference between Black Drop and the default? Let's remove one to be less confusing.